src/main/resources/rules/rulesDefault.json

```diff
@@ -164,6 +171,7 @@
     }
   },
   {
+    "id": "instance.identifiers.systemcontrolnumber",
```

Comment:
`instance.identifiers.system.control.number`
If you look at transformationfields API this is generated as 
`{
      "fieldId": "instance.identifiers.system.control.number",
      "displayNameKey": "instance.identifiers",
      "referenceDataValue": "System control number",
      "path": "$.instance[*].identifiers[?(@.identifierTypeId==7e591197-f335-4afb-bc6d-a6d76ca3bace)].value",
      "recordType": "INSTANCE"
    },`